### PR TITLE
Exosuit Upkeep

### PR DESCRIPTION
**Pull request by Mechoid**
Exosuit upkeep, lower slowdown of Durand and Gygax components, guns now slow far less.

### CODE REVIEW

File: polaris.dme [hunk 1]
```diff
@@ -1872,7 +1872,6 @@
 #include "code\modules\hydroponics\grown_predefined.dm"
 #include "code\modules\hydroponics\seed.dm"
 #include "code\modules\hydroponics\seed_controller.dm"
-#include "code\modules\hydroponics\seed_datums.dm"
```
Comment (listerla) [inline polaris.dme:1875]: is there a reason we're executing seed_datums

Comment (Mechoid) [reply polaris.dme:1875]: ..God damnit branch swap

Comment (Atermonera) [reply polaris.dme]: I mean that code _does_ need to die in the fieriest pits of hell available...

Comment (Mechoid) [reply polaris.dme]: https://github.com/PolarisSS13/Polaris/pull/7475

**Comment by FenodyreeAv** [comment]
\o/

**Comment by Atermonera** [comment]
Staff input: going with this instead of #7476 and #7471, and then we'll evaluate the results at a later date.

**Pull request merged**